psi4/src/psi4/libmints/oeprop.h

```diff
@@ -278,7 +278,7 @@ class MultipolePropCalc : public Prop {
     /// Common initialization
     MultipolePropCalc(std::shared_ptr<Wavefunction> wfn, Vector3 const& origin);
     // Output Type of multipole function, name, elec, nuc, tot
```

Comment:
The int below is undocumented.

Reply:
good point -- I'll stick `and order` into the next commit.